components/11-base/fonts/templates/_icons.template

```diff
@@ -25,6 +25,7 @@ $id: random(1000);
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   font-style: normal;
+  font-size: 24px;  // fixed size for all icons
```

Comment:
Comment mag weg hier.
Andere comments schrijven we altijd volgens conventie met hoofdletter en leesteken.